src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -381,6 +390,13 @@ private String route(AppDeploymentRequest request) {
 		return request.getDeploymentProperties().get(ROUTE_PROPERTY);
 	}
 
+    private Set<String> routes(AppDeploymentRequest request) {
+        Set<String> routes = new HashSet<>();
+        routes.addAll(this.deploymentProperties.getRoutes());
+        routes.addAll(StringUtils.commaDelimitedListToSet(request.getDeploymentProperties().get(SERVICES_PROPERTY_KEY)));
```

Comment:
Is there a reason why `services` property key is associated here?

Reply:
Oops, used this as a reference:
https://github.com/spring-cloud/spring-cloud-deployer-cloudfoundry/blob/67d5907095ea5f95e58e8695f4cae559028c9ab7/src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java#L72-L77

and failed to update the key